src/SQLChecks/Functions/Public/Get-ManagedBackupDatabasesWithoutEncryption.ps1

```diff
@@ -27,11 +27,7 @@ Function Get-ManagedBackupDatabasesWithoutEncryption {
     and     [db_name] not in ('master','model','tempdb');
 "@
 
- 
-    Invoke-Sqlcmd -ServerInstance $serverInstance -query $query -Database msdb | Where-Object {
-        $ExcludeDatabases -notcontains $_.DatabaseName } | ForEach-Object {
-        [PSCustomObject]@{
-            DatabaseName = $_.DatabaseName
-        }
-    }
+    Invoke-Sqlcmd -ServerInstance $serverInstance -Query $query -Database msdb |
```

Comment:
Database context is redundant here since the query 3-part-qualifies the target object.

```suggestion
    Invoke-Sqlcmd -ServerInstance $serverInstance -Query $query |
```

Reply:
I don't mind the redundancy. Correct connection is IMHO better than a 3-part name